### DISCUSSION TITLE
Adding Librivox

### DISCUSSION THREAD

**Discussion by abhyudaya633** [PaulWoitaschek/Voice#1296] [Ideas]
Ability to download audiobooks from Librivox. It will increase the initial appeal of the app by providing a large repository of free audiobooks.

(similar feature has been implemented by Moon+reader for downloading ebooks from Project Gutenberg)

**Comment by ahstro** [comment]
Maybe a compromise, if people don't want to add the feature to the app itself, would be to mention LibriVox on the initial screen, when there are no audiobooks added?

**Comment by abhyudaya633** [comment]
I do not understand why people are against this idea. Is it because this will ruin the app or this is technically unfeasible?
